src/khoj/database/migrations/0029_userrequests.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.7 on 2024-01-28 12:38
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("database", "0028_khojuser_verified_phone_number"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="UserRequests",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                ("request", models.CharField(max_length=200)),
```

Comment:
What is the purpose of the `request` Char field and the `updated_at` DateTimeField ? They don't seem to be used anywhere

Reply:
`created_at` and `updated_at` are added to all of our models and managed independently. But you're right, the `request` field is superfluous. Will remove that.